src/states/elder/mod.rs

```diff
@@ -458,10 +481,59 @@ impl Elder {
         CompleteParsecReset {
             gen_pfx_info,
             to_vote_again,
+            to_process,
             event_to_send: None,
         }
     }
 
+    fn process_post_reset_events(
+        &mut self,
+        _old_pfx: Prefix<XorName>,
+        to_process: BTreeSet<NetworkEvent>,
+    ) {
+        to_process.iter().for_each(|event| match &event.payload {
+            evt @ AccumulatingEvent::Offline(_)
+            | evt @ AccumulatingEvent::AckMessage(_)
+            | evt @ AccumulatingEvent::StartDkg(_)
+            | evt @ AccumulatingEvent::ParsecPrune
+            | evt @ AccumulatingEvent::Relocate(_)
+            | evt @ AccumulatingEvent::SectionInfo(_, _)
+            | evt @ AccumulatingEvent::NeighbourInfo(_)
+            | evt @ AccumulatingEvent::TheirKeyInfo(_)
+            | evt @ AccumulatingEvent::SendAckMessage(_)
+            | evt @ AccumulatingEvent::User(_) => {
+                log_or_panic!(LogLevel::Error, "unexpected event {:?}", evt);
+            }
+            AccumulatingEvent::Online(payload) => {
+                self.resend_bootstrap_response_join(&payload.p2p_node);
```

Comment:
Shouldn't this only happen on split and not on other parsec reset? Otherwise would we be unnecessarily resending the `BootstrapResponse`?

Reply:
Any time churn happens, but not on parsec reset yes, I may have gotten that wrong

Reply:
Why does it need to happen on non-split churn?

Reply:
Ah, is it because some elders might get demoted and they would not re-vote it and so it would not accumulate?

Reply:
Yes